src/sparql/rm_xref_by_prefix.ru

```diff
@@ -16,5 +16,5 @@ WHERE {
     oboInOwl:hasDbXref ?xref ;
     ?p1 ?o2 .
   }
-  FILTER( STRSTARTS(str(?xref), "UMLS_ICD9CM_2005_AUI:") || STRSTARTS(str(?xref), "ICD11:") || STRSTARTS(str(?xref), "SNOMEDCT_US_") || STRSTARTS(str(?xref), "IMDRF:") || STRSTARTS(str(?xref), "url:") )
```

Comment:
Just commenting on my understanding.
I see what's going on here to address #140. So this deletion query gets run on the DOID and NCIT components, removing these xrefs. You're removing ICD11 from this query, so thus we're adding the mappings back from these files, as they won't be removed any longer.

I thought that #140 had something to do with my new ICD11Foundation work but it does not.